R/unique-osmdata.R

```diff
@@ -1,15 +1,15 @@
 #' unique_osmdata
 #'
 #' Reduce the components of an \link{osmdata} object in 'sf' or 'sp' form (that
-#' is, obtained from \link{osmdata_sf} or \link{osmdata_sp}) to only unique
+#' is, obtained from [osmdata_sf()] or [osmdata_sp()] to only unique
```

Comment:
```suggestion
#' is, obtained from \link{osmdata_sf} or \link{osmdata_sp} to only unique
```

Reply:
\link{fun} is equivalent to [fun()] in the .Rd file and is [expanded by roxygen](https://roxygen2.r-lib.org/articles/markdown.html#links) (see .Rd files in the PR)